### ISSUE TITLE
Problems with FormMultiCheckBoxElement

### ISSUE THREAD

**Issue by dhaksddj** [TheJuki/KFormMaster#27]
## Description

I do not see the observers firing when the user has chosen their options.

I'm declaring the element like this:

FormPickerMultiCheckBoxElement<String>()
                            .setOptions(listOf("A", "B", "C"))
                            .addValueObserver{ value, element ->
                                Timber.d("Hello world!!! value: %s", value)
                            }
### Expected behavior

the value observer to fire

### Actual behavior

Nothing happens
### Steps to reproduce the bug

1. [First step]
2. [Second step]
3. [and so on...]

### KFormMaster version

* Version: 3.2.0

**Comment by TheJuki** [comment]
Yep. I forgot to fix that when I realized the value observer only fires when the value changes.  FormPickerMultiCheckBoxElement uses the optionsSelection variable to store the options selected. Once I fix that I will need to mention in the docs to not use the value property:

```
 FormPickerMultiCheckBoxElement()
.setOptions(listOf("A", "B", "C"))
.addValueObserver{ _, element ->
Timber.d("Hello world!!! value: %s", element.optionsSelected)
}
```

**Comment by dhaksddj** [comment]
Thats great, thank you. Any idea when you might have a fix?

PS Good job on the documentation.

**Comment by TheJuki** [comment]
Thanks! The valueObservers are not being fired currently. Since it is a small fix I should have it done tonight and release a new minor version.

**Comment by dhaksddj** [comment]
Thanks so much!

**Comment by TheJuki** [comment]
I just released [4.0.0](https://github.com/TheJuki/KFormMaster/releases/latest). Please update your dependency and let me know if valueObservers works for you. Please note that optionsSelected has been removed. Value should now have the options selected. 

```
FormPickerMultiCheckBoxElement<List<String>>()
.setOptions(listOf("A", "B", "C"))
.addValueObserver{ value, element ->
Timber.d("Hello world!!! value: %s", value)
}
```

**Comment by dhaksddj** [comment]
I can confirm the new changes seem to work, the listener is firing and being passed the appropriate choices.

Thanks for the speedy response!
